packages/p/polylith/files/poly

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+ARGS=""
+while [ "$1" != "" ] ; do
+ ARGS="$ARGS $1"
+ shift
+done
+
+exec "java" $JVM_OPTS "-jar" "/usr/lib/polylith/poly.jar" $ARGS
```

Comment:
This is still going to have the same issue of not working if JAVA_HOME isn't set. Please take a look at the shim script located at `packages/d/davmail/files/davmail` to see how this is usually done. Basically, check to see if JAVA_HOME is defined, if not define it to the default and use it.

Reply:
updated the shell script.